### ISSUE TITLE
How to enter value in 12 hour format?

### ISSUE THREAD

**Issue by rhap5ody** [wojtekmaj/react-time-picker#42]
Hi. I want to enter value in 12 hour format but the app crashes saying i didn't provide a valid value for the `value` prop. How do I do that? Here is an example i try to use:

```
<TimePicker
  value='06:00 PM'
  clockIcon={null}
  disableClock
  format='hh:mm a'
/>
```

Thanks.

**Comment by emilyuhde** [comment]
The time passed into the `value` prop needs to be in 24 hour format and the TimePicker will handle converting the way it's displayed if your locale or format specifies it should be a 12 hour format time. So you should set `value='18:00'`.

**Comment by emilyuhde** [comment]
Also, I also disabled the clock in my usage nad I only had to set `disableClock`. `clockIcon` was automatically taken care of so I didn't need to set it to `null`.

**Comment by rhap5ody** [comment]
@emilyuhde I get the value from api in 12 hour format so I want to use it directly as such. Isn't there an option to enter it in 12 hour format and also display it in it? Thanks

**Comment by emilyuhde** [comment]
You could convert the time you get from the API into a Date object before passing it into the `value` prop, but from what I've seen, it's still going to return to you a string in 24 hour format from the `onChange`.

**Comment by jpcmf** [comment]
Is possible use as 24 hour format? I mean, just enter 22:00?

**Comment by wojtekmaj** [comment]
Yes, of course. By default, time picker format is based on detected user browser's locale, unless `locale` prop is provided (e.g. `'de-DE'` will give you 24-hour clock, and `'en-US'` - 12-hour clock). And this behavior can also be overwritten using `format` prop.
